web-share/src/svelte/components/Media.svelte

```diff
@@ -96,4 +106,7 @@ limitations under the License.
       <Button handleClick={handleDelete}>Delete</Button>
     </div>
   {/if}
+  {#if showSize}
+    <div class="size">{prettyBytes(size)}</div>
```

Comment:
FYI, even though your web app says 1.98 MB, Chrome Downloads says 0B.

Reply:
Yeah, the folks responsible for Chrome Downloads are aware that it's always going to be `0B` there. I think they might remove it from the Chrome UI?

Not much to do about that here, though.